app/create_file.py

```diff
@@ -1 +1,56 @@
-# write your code here
+from datetime import datetime
+from sys import argv
+from os import mkdir, chdir
+
+filename = ""
+directories = []
+
+if "-f" in argv:
```

Comment:
Simplify the code structure, use this code instead of your if-else blocks here:
```
# Check for "-f" flag to specify filename
if "-f" in argv:
    index = argv.index("-f") + 1
    filename = argv[index]

# Check for "-d" flag to specify directories
if "-d" in argv:
    index = argv.index("-d") + 1
    directories = argv[index:]

# If no filename provided, use default "file.txt"
if not filename:
    filename = "file.txt"

```